src/index.html

```diff
@@ -22,6 +22,323 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!-- HEADER -->
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+
+      <nav class="navigation">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link nav_link--is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="index.html"
```

Comment:
All navigation links point to 'index.html'. If these links are intended to navigate to different sections or pages, consider updating the 'href' attributes to the correct URLs.